packages/oasis-actions/src/strategies/aave/depositBorrow.ts

```diff
@@ -9,8 +9,9 @@ import * as operations from '../../operations'
 import { AAVEStrategyAddresses } from '../../operations/aave/addresses'
 import { BorrowArgs } from '../../operations/aave/borrow'
 import { DepositArgs } from '../../operations/aave/deposit'
-import { Address, IPositionTransition, IPositionTransitionDependencies, SwapData } from '../types'
+import { IPositionTransition, SwapData } from '../types'
 import { AAVETokens } from '../types/aave'
+import { Address, IPositionTransitionDependencies } from '../types/StrategyParams'
```

Comment:
The concept of the one `index.ts` in the `types` folder is beneficial because we don't need to change all imports after changing the file name in this example.

Reply:
Yea, everything is exported from an index. Just think there's a few old import lines hanging around. I probably updated that one with a find/replcae